src/components/rmrk/Gallery/CollectionItem.vue

```diff
@@ -13,7 +13,10 @@
             ></b-image>
           </div>
           <h1 class="title is-2">
-            {{ name }}
+            <template v-if="!nameLoading">
+              {{ name }}
+            </template>
+            <b-skeleton :active="nameLoading" :height="30" size="is-medium" :animated="true"></b-skeleton>
```

Comment:
```suggestion
            <b-skeleton :active="nameLoading" size="is-large"></b-skeleton>
```
- animated property is by default true
- I would choose between using heigh or size but not both

Reply:
Pushed the fix